web/Controllers/PropertyController.cs

```diff
@@ -123,4 +124,21 @@ public ActionResult AddNewProperty_Cancel()
         // Go to View Properties page
         return RedirectToAction("ViewProperties", "Landlord");
     }
-}
+
+    [HttpGet]
+    [Route("/property/view")]
+    public ActionResult ViewProperty(int propertyId)
+    {
+        PropertyDbModel? model = _propertyService.GetPropertyByPropertyId(propertyId);
+        if (model == null)
+        {
+            _logger.LogWarning("View property did not return the property as expected");
+            return RedirectToAction(nameof(LandlordController.ViewProperties), "Landlord");
+        }
+        else
+        {
+            PropertyViewModel propertyViewModel = PropertyViewModel.FromDbModel(model);
+            return View(propertyViewModel);
+        }

```

Comment:
Doesn't need to be in an `else` block because of the early return above

Reply:
yeah sure

Reply:
ok